internal/pkg/cli/svc_deploy.go

```diff
@@ -358,6 +373,23 @@ func (o *deploySvcOpts) configureClients() error {
 	return nil
 }
 
+func (o *deploySvcOpts) publicCIDRBlocks() ([]string, error) {
+	envDescription, err := o.envDescriber.Describe()
+	if err != nil {
+		return nil, fmt.Errorf("describe environment %s: %w", o.envName, err)
+	}
+	vpcID := envDescription.EnvironmentVPC.ID
+	subnets, err := o.subnetLister.ListVPCSubnets(vpcID)
+	if err != nil {
+		return nil, fmt.Errorf("list VPC subnets of the environmnent %s: %w", o.envName, err)
```

Comment:
Can we include the vpc id in the error as well? (and small typo)
```suggestion
		return nil, fmt.Errorf("list subnets of vpc %s in environment %s: %w", vpcID, o.envName, err)
```